src/Api/Transaction/Start.php

```diff
@@ -253,7 +253,7 @@ public function addProduct(
      *  language
      *  accessCode
      *  gender (M or F)
-     *  dob (DD-MM-YYYY)
+     *  birthDate (DD-MM-YYYY)
```

Comment:
This should be dob

Reply:
Fair enough; that's pretty confusing to be honest.